src/Apps/Sell/Components/ImagePreviewItem.tsx

```diff
@@ -98,6 +98,8 @@ export const ImagePreviewItem: React.FC<ImagePreviewItemProps> = ({
                 objectFit: "contain",
               }}
               onError={() => {
+                if (!photoSrc.startsWith("data:")) return
```

Comment:
We only want to show the "Processing..." message when the uploaded file itself cannot be displayed. When the image comes from Gemini and cannot be loaded, probably something else is wrong.